packages/parser-javascript/src/parser.ts

```diff
@@ -34,6 +105,34 @@ export default class JavascriptParser extends WebhintParser<ScriptEvents> {
                 tokens,
                 walk
             });
+
+            Object.entries(walkArrays).forEach(([methodName, walkArray]) => {
+                walkArray.forEach(([{ node, state, base }, visitors]) => {
+                    let fullVisitors: any | Function = {};
```

Comment:
What about `allVisitors`?
```suggestion
                    let allVisitors: any | Function = {};
```